DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts.cshtml

```diff
@@ -67,26 +67,35 @@
     Contacts at the trust
   </h2>
   <div class="govuk-summary-card" data-testid="accounting-officer">
-    <div class="govuk-summary-card__title-wrapper">
+    <div class="govuk-summary-card__title-wrapper contact-card-title-with-disclaimer">
       <h3 class="govuk-summary-card__title">
         Accounting officer
       </h3>
+      <div class="govuk-summary-card__actions govuk-!-margin-0">
+        @{ DisplayDisclaimerText(); }
```

Comment:
Using @ and then a method is primarily for small utility, not rendering HTML. We should be using partials or taghelpers for this. 

In this instance a partial is most appropriate

_ContactsDisclaimerText.cshtml
``` 
<div class="govuk-warning-text govuk-!-margin-bottom-0">
  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
  <strong class="govuk-warning-text__text">
    <span class="govuk-visually-hidden">Warning</span>
    Email address for internal use only
  </strong>
</div>
```
 and you would reference it as such
 ```
 @await Html.PartialAsync("_ContactsDisclaimerText")
 ```

Reply:
Or you could make it _CardDisclaimerText provided it would be reusable for future cards